x-pack/plugins/lens/public/xy_visualization/xy_suggestions.ts

```diff
@@ -222,8 +237,21 @@ function getSuggestionsForLayer({
   mainPalette?: PaletteOutput;
```

Comment:
One input that you might be able to use here is `changeType`: we can have different suggestions for different types of changes, such as `initial` vs `extended`.